CMakeLists.txt

```diff
@@ -183,3 +183,7 @@ include( CTest )
 if( BUILD_TESTING AND TRACCC_BUILD_TESTING )
    add_subdirectory( tests )
 endif()
+
+if(TRACCC_BUILD_FUTHARK)
```

Comment:
Let's declare a proper cache variable with this. Like:

```cmake
option( TRACCC_BUILD_FUTHARK "Build the Futhark code included in traccc" FALSE )
```

You should put this around line 50, with the other similar "options".